tests/regression/test_regression.py

```diff
@@ -96,17 +101,54 @@ def setup_tearndown():
 
     # download regression artifacts from Hugging Face Hub at the start
     snapshot_download(repo_id=HF_REPO, local_dir=REGRESSION_DIR)
+
+    # WARNING: If running on XPU, LORA_4BIT_FOLDER artifacts are loaded from HF_REPO_XPU, which is outside of
+    # peft direct control. The load_output function uses torch.load, which can execute arbitrary
+    # code from pickle files. Users should be aware of this potential security risk.
+    is_xpu = (infer_device() == "xpu")
+    if is_xpu:
+        lora_4bit_folder_path = os.path.join(REGRESSION_DIR, LORA_4BIT_FOLDER)
+        if os.path.isdir(lora_4bit_folder_path):
```

Comment:
Since the `REGRESSION_DIR` is a temp folder created on script start, this shouldn't be necessary, right?